src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfirm.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, Card, CardBody, CardFooter, CardHeader, Heading } from '@chakra-ui/react'
+import { useCallback } from 'react'
+import { useHistory } from 'react-router'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+
+import { WithBackButton } from '../../WithBackButton'
+import { LimitOrderRoutePaths } from '../types'
+
+const cardBorderRadius = { base: 'xl' }
+
+export const LimitOrderConfirm = () => {
+  const history = useHistory()
+
+  const handleBack = useCallback(() => {
+    history.push(LimitOrderRoutePaths.Input)
+  }, [history])
+
+  const handleConfirm = useCallback(() => {
+    history.push(LimitOrderRoutePaths.Status)
+  }, [history])
+
+  return (
+    <SlideTransition>
+      <Card
+        flex={1}
+        borderRadius={cardBorderRadius}
+        width='full'
+        variant='dashboard'
+        maxWidth='500px'
+      >
+        <CardHeader px={6} pt={4}>
+          <WithBackButton onBack={handleBack}>
+            <Heading textAlign='center' fontSize='md'>
+              <Text translation='limitOrder.confirm' />
```

Comment:
Shouldn't we add all those copies to the main.json file also at this point?

Reply:
Absolootely yes, will do later though because all of this is about to change pending conversation with product 🙏